features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Shared/Onboarding/OnboardingMenuCard.jsx

```diff
@@ -106,11 +107,20 @@ const RestAPIMenu = (props) => {
                                 alignItems='center'
                                 pt={15}
                                 justifyContent='center'
-                                color='primary.main'
+                                color={disabled ? 'disabled' : 'primary.main'}
                             >
                                 {name}
                             </Box>
                         </Grid>
+                        <Grid item xs={11}>
+                            {disabled && (
+                                <Box>
+                                    <Typography variant='body2' color='primary'>
+                                        *You are not authorized to create/update due to insufficient permissions
```

Comment:
This message looks sketchy 
my suggestions
```suggestion
                                        *You are not authorized to create or update scopes due to insufficient permissions
```

Reply:
Fixed in fe10341